tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -180,7 +180,8 @@ func ExecSignAndBroadcast(
 	io.Println(string(bres.DeliverTx.Data))
 	io.Println("OK!")
 	io.Println("GAS WANTED:", bres.DeliverTx.GasWanted)
-	io.Println("GAS USED:  ", bres.DeliverTx.GasUsed)
+	io.Println("GAS USED:", bres.DeliverTx.GasUsed)
+	io.Println("EVENTS:", string(bres.DeliverTx.EncodeEvents()))
```

Comment:
There is excessive escaping in this generated output:

```json
["{\"type\":\"sender\",\"pkg_path\":\"gno.land/r/demo/cbee\",\"identifier\":\"subSender\",\"timestamp\":1713769028,\"attributes\":[{\"key\":\"k1\",\"value\":\"v1\"},{\"key\":\"k2\",\"value\":\"v2\"}]}","{\"type\":\"receiver\",\"pkg_path\":\"gno.land/r/demo/cbee\",\"identifier\":\"subReceiver\",\"timestamp\":1713769028,\"attributes\":[{\"key\":\"bar\",\"value\":\"baz\"}]}"]
```

It is technically valid JSON, but not usable. It should be both valid and usable JSON, like this:

```json [{"type":"sender","pkg_path":"gno.land/r/demo/cbee","identifier":"subSender","timestamp":1713769028,"attributes":[{"key":"k1","value":"v1"},{"key":"k2","value":"v2"}]},{"type":"receiver","pkg_path":"gno.land/r/demo/cbee","identifier":"subReceiver","timestamp":1713769028,"attributes":[{"key":"bar","value":"baz"}]}]
```

Reply:
Not sure where this was fixed (which commit), but it was fixed 👍